fearless_simd/src/lib.rs

```diff
@@ -20,7 +20,7 @@
 //! To call a function with the best available target features and get the associated `Simd`
 //! implementation, use the [`dispatch!()`] macro:
 //!
-//! ```rust
+//! ```rust,ignore
```

Comment:
Does this put the doctests at risk of bitrotting? Why do they break the build?

Reply:
Augh, figured out why the doctests didn't pass. `RUSTFLAGS` and `RUSTDOCFLAGS` both need to be set to the exact same values, or weird things happen (https://github.com/rust-lang/rust/issues/134172, see also https://github.com/rust-lang/cargo/issues/12739 and [this IRLO discussion thread](https://internals.rust-lang.org/t/can-we-do-something-about-rustflags-rustdocflags/21513)). After setting `target.wasm32-wasip1.rustdocflags` in the test config, things work.